ui/js/actions/app.js

```diff
@@ -297,6 +297,14 @@ export function doClearCache() {
   };
 }
 
+export function doChangeLanguage(newLanguage) {
+  app.i18n.setLocale(newLanguage);
+  return {
```

Comment:
I guess we could leave it there in case there are some actions that need to be performed in the future upon language change. @filipnyquist Any thoughts on this?